__tests__/submissionService.test.ts

```diff
@@ -0,0 +1,75 @@
+import { expect } from '@jest/globals'
+import resolvers from '../src/resolvers'
+import loadSchema from '../src/schema'
+import request from 'supertest'
+import { ApolloServer } from '@apollo/server'
+import { startStandaloneServer } from '@apollo/server/standalone'
+import { initiatilizeFirebaseInstance } from '../src/firebaseDb'
+
+const queryData = {
+    query: `mutation Mutation($input: SubmissionInput) {
+        createSubmission(input: $input) {
+          id
+          googleMapsUrl
+          createdDate
+          healthcareProfessionalName
+          isApproved
+          isRejected
+          isUnderReview
+          spokenLanguages {
+            iso639_3
+            nameEn
+            nameJa
+            nameNative
+          }
+          updatedDate
+        }
+      }`,
+    variables: {
+        input: {
+            googleMapsUrl: 'http://domain.com',
+            healthcareProfessionalName: 'some name',
+            isApproved: true,
+            spokenLanguages: [
+                {
+                    iso639_3: 'en',
+                    nameEn: 'English',
+                    nameJa: 'Eigo',
+                    nameNative: 'English'
+                }
+            ]
+        }
+    }
+}
+
+describe('newSubmission', () => {
+    let url: string
+
+    const server = new ApolloServer({
+        typeDefs: loadSchema(),
+        resolvers
+    })
+
+    beforeAll(async () => {
+        ({ url } = await startStandaloneServer(server, {listen: { port: 0 }}))
+        await initiatilizeFirebaseInstance()
```

Comment:
@ermish Do you know how we could stop the instance between each test or clear the emulator data?